Fixes #5460 - Add Manchester (EGCC) Ground Network

Hinshee opened the pull request:
Fixes #5460 
With thanks to Dean Benavidez 🙏
# Summary of changes

- Ground Network Added
- Updated Runway Exit Order

# Screenshots (if necessary)

![EuroScope_lWovPx2fk9](https://github.com/user-attachments/assets/f66a717d-93f8-4d1e-b147-e37dc2dc87b5)

BenWalker01 commented:
![image](https://github.com/user-attachments/assets/ea9ffabe-44c4-4303-82e1-f2c439adcd73)
I think this needs updating for the latest airac (2410)

AliceFord commented:
Unrelated, but is there a Heathrow ground network anywhere @Hinshee ?

Hinshee commented:
> Unrelated, but is there a Heathrow ground network anywhere @Hinshee ?
Yes! But somewhere. I'll ask around.

AliceFord commented:
@Hinshee Me and peter found one that Sam James made a while ago, we'll upload it in a separate PR!

Hinshee commented:
![EuroScope_1F2xuxZT11](https://github.com/user-attachments/assets/34192453-2de7-4e99-aff6-d66209517607)
Changelog:
- Added Stands 113L, 113R, 116L, 112R, 110L, 106R, 55L and 55R
- Reworked E / NA and Z Taxi Area
- Added Exit P1 and F1 to Runway 05L
- Added VA1 and T1 to Runway 05R

BenWalker01 reviewed (changes requested):
@Hinshee Still looks like the old Z taxi line. As of 2410 Z orange / blue has been extended down towards Z1. Compiled SF [here](https://github.com/VATSIM-UK/UK-Sector-File/actions/runs/10976147500/artifacts/1962222271)
Also might be worth checking that the M loop works with sb files.

Hinshee commented:
> Still looks like the old Z taxi line. As of 2410 Z orange / blue has been extended down towards Z1.

Gotcha now! That would explain my confusion 🤣 

>  Also might be worth checking that the M loop works with sb files.
> Getting them to 'line-up' using after having entered at M, backtracked, and used the turning circle at full length of 23R didn't work, as they weren't pointing at the centreline defined in the SB Scenerio we were testing with. So it might be worth considering having the turning circle taxi end slightly early so they definitely point towards the centreline and can line-up/take-off one taxied there. (I suspect this is unlikely to get used a great deal, but since you've gone to the effort of adding it :) )

Wasn't my intention to add it, and if I'm quite honest, I can't see it being used.
I'll reconsider if it's worth having it in or not.

Shall do a final test with the new TWR sweatboxes soon.

PLM1995 reviewed (changes requested):
This still doesn't looks correct to me, I would have thought you at a minimum want the Z blue to follow the updated Z blue from the new Z1 position through Z2 and onwards past stand 919. At present, anything on Z blue north of 919 will have to 'cross' randomly onto Z orange as it passes 919.
![image](https://github.com/user-attachments/assets/ab30776a-1f60-4ba5-ac6a-8b7f008e41cb)

PLM1995 commented:
> I named the recent push WIP since I’m not finished… I’ll see to finishing it soon - University comes first.

Oh sorry I didn't spot that! Yes of course, no worries :)

PLM1995 reviewed (approved):
Looks good! Thanks @Hinshee - great piece of work :)

BenWalker01 reviewed (approved):
Looks good :)
Functionally works, but we were wondering is there a reason the crossing taxiways are not defined?

Hinshee commented:
> Looks good :) Functionally works, but we were wondering is there a reason the crossing taxiways are not defined?

Could you give me an example? @BenWalker01

BenWalker01 commented:
Sorry. Should have been more clear. The taxiways that cross 23R/05L, for example P1 to DZ1. 
I'm not particularly sure how the sweatbox works, or if having those defined breaks arriving traffic but when I was crossing I had to manually draw to the other side of the runway before rejoining the network.
